.circleci/config.yml

```diff
@@ -5,11 +5,19 @@ jobs:
     docker:
       - image: circleci/rust:1.41-stretch
     steps:
-      - run: rustup toolchain install nightly
+      - checkout
```

Comment:
👆needs to be done first, such that rustup finds the file `./rust-toolchain` from the repo's root